### ISSUE TITLE
[Bug] Wrong Emoji. Red* Heart.

### ISSUE THREAD

**Issue by tobealive** [linuxmint/cinnamon-spices-applets#4080]
**Pictures tell more then words:**
![heartemoji](https://user-images.githubusercontent.com/34311583/144743325-2dbb1188-c96f-4b6f-a23d-07347386bc3a.jpg)

**Comment by tobealive** [comment]
Using Manjaro Cinnamon.
Cinnamenu Version 4.14.15.

Probably Cinnamenu doesn't even has his own emojis. Then this could be an issue with with noto-fonts-emoji or something.

**Comment by fredcw** [comment]
Hi @tobealive, sorry I didn't reply earlier but I only just spotted your post. Thanks for reporting this, it will be fixed in the next update. I also have noto emoji font but it appears as a red heart for me so it must be something to do the system font config. Cinnamenu was just using U+2764 (❤) which is supposed to be a "heavy black heart" but appears as a red heart in many places. I've now changed it to U+2764️ U+FE0F which should make sure it's a red heart on all systems.
